pybamm/discretisations/discretisation.py

```diff
@@ -181,6 +184,35 @@ def process_dict(self, var_eqn_dict):
             # keep calling .id
         return var_eqn_dict
 
+    def process_list(self, eqn_list):
+        """Discretise a list of equations, broadcasting if necessary (for model.events).
```

Comment:
why do the events need to be broadcasted? Presumably you only want a single number (ie. the minimum concentration)?